site/app/templates/grading/electronic/Details.twig

```diff
@@ -64,14 +64,23 @@
             var button = document.getElementById('inquiryButton');
             status = Cookies.get('inquiry_status')
             if (status === 'on') {
-                grade_inquiry_only();
+                $('[data-testid=grade-button]').each(function() {
+                    const hasGradeInquiry = typeof $(this).attr('data-grade-inquiry') !== 'undefined';
+                    if (!hasGradeInquiry) {
```

Comment:
It might be simpler to inline `hasGradeInquiry` here and get rid of the temporary variable.